custom/icds_reports/data_pull/sql_queries/cbe_conducted.sql

```diff
@@ -0,0 +1,15 @@
+SELECT district_name,
```

Comment:
didn't review these SQL files closely

Reply:
thats fine, these queries are just copied over from previous runs, with the only change being adding the month as parameter instead.

Reply:
thats fine, these queries are just copied over from previous runs, with the only change being adding the month as parameter instead.